CHANGES.md

```diff
@@ -1,6 +1,417 @@
+* added checks for division by 0 to `bigint` division methods  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/BigInteger.html#BigInteger.bigint.divexact,  
+  https://chapel-lang.org/docs/1.26/modules/standard/BigInteger.html#BigInteger.bigint.divQ,  
+  https://chapel-lang.org/docs/1.26/modules/standard/BigInteger.html#BigInteger.bigint.divR,  
+  https://chapel-lang.org/docs/1.26/modules/standard/BigInteger.html#BigInteger.bigint.divQR,  
+  https://chapel-lang.org/docs/1.26/modules/standard/BigInteger.html#BigInteger.bigint./,  
+  and https://chapel-lang.org/docs/1.26/modules/standard/BigInteger.html#BigInteger.bigint./=)
+
+Bug Fixes
+---------
+* fixed a compiler bug causing incorrect results for dynamic dispatch
+* fixed several problems with error handling in the context of parallel loops
+* fixed a bug preventing `const in` task intents from being used with arrays
+* fixed a bug when calling `.localSlice()` on a slice of a default array
+* fixed a bug in which `require "M.chpl";` was ignored within implicit modules 
+* fixed a potential issue with arbitrary shell commands in `require` statements
+* fixed a bug in which first-class functions failed to retain return intents
+* fixed a problem with combinations of `const ref`, `owned`, and `coforall`
+* fixed the naming of `CHPL_` variables in the output of `chpl --help-env`
+* fixed a bug that caused sporadic problems  with the `--memLog` flag
+* added a remote cache fence to barrier calls
+* fixed a bug with overload sets and operators
+
+Bug Fixes for Build Issues
+--------------------------
+* enabled building with GPU support for host compilers other than `clang`
+
+Bug Fixes for Libraries
+-----------------------
+* fixed a bug with `bigint.removeFactor()` when `fac` is `0`
+
+Third-Party Software Changes
+----------------------------
+* updated the bundled copy of LLVM and clang to version 13
+* updated the bundled copy of `libfabric` to version 1.13.2
+* updated the version of Sphinx used for `chpldoc` to 4.3.2
+* updated the Sphinx Chapel domain version used for `chpldoc` to 0.0.21
+* updated the breathe version used for `chpldoc` to 4.31.0
+
+Developer-oriented changes: Process
+-----------------------------------
+* trailing whitespace is now flagged in CI using GitHub actions
+
+Developer-oriented changes: Documentation
+-----------------------------------------
+* removed references to outdated files from the `chplspell` documentation
+* fixed the path to 'ChapelIO' in 'How To Generate Warnings And Error Messages'
+
+Developer-oriented changes: Naming Changes
+------------------------------------------
+* coined the name 'dyno' to refer to aspects of the compiler rewrite effort
+
+Developer-oriented changes: Module changes
+------------------------------------------
+* simplified the barrier used to initialize locales
+* updated the 'Socket' module's operators to use the `operator` keyword
```

Comment:
I think socket is new this release, so no need to mention bug fixes to it?

Reply:
Oh, good catch—point Ronaghan!